### ISSUE TITLE
Workaround BZ #1393034 by associating fake capsules after creation

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#4369]
BZ https://bugzilla.redhat.com/show_bug.cgi?id=1393034 affects our automation badly.
It is about that you cannot set locations/organizations upon capsule creation.
Furthermore a capsule is always created associated to ALL organizations/locations.

But if it is a fake capsule and a test tears it down, then all organizations/locations are affected by non-communicating capsule. Affecting further testing.

To minimize these bad effects lets update capsule right after its creation and associate it only with relevant organization/location in ```make_capsule()```

...then only a small portion of tests will be affected. (time between creation and update is much lower than time between creation and teardown) 

(Or make devs fix it which may turn futile)

**Comment by rplevka** [comment]
okay, we got some news on this and it is a real fun:
https://bugzilla.redhat.com/show_bug.cgi?id=1393034#c5

tl;dr:
satellite recognizes the internal capsule by detecting a `pulp` feature and treats it a special way - assigns it to all the orgs&locs.
So the parameters are probably overridden.

- looks like we might need a real capsule in place and perform the port forwarding there,

**Comment by ldjebran** [comment]
@lpramuk @rplevka , I implemented a real capsule https://github.com/SatelliteQE/robottelo/pull/4381 and this behavior is not noticeable, I believe that  @rplevka your comment in bugzilla bug page is actual, any way we can now even automate that bug with a real capsule_node, as with capsule infra implementation there is a test  cli/test_contentview.test_positive_remove_cv_version_from_multi_env_capsule_scenario
that is passing and you can see the logs of all the tests there: https://github.com/SatelliteQE/robottelo/pull/4381

**Comment by rplevka** [comment]
@ldjebran we discussed this with @lpramuk and came to the conclusion, that the bug is still valid as the system allows user to modify (even the internal) capsule so it only belongs to a subset of orgs&locs. What's more, it is a valid scenario to have an internal capsule only exposed to some orgs.

I remember i posted the comment to the BZ but can't see it there...so i will repost this.

**Comment by rplevka** [comment]
closing this one in favor of
https://github.com/SatelliteQE/robottelo/issues/4577
